### PR TITLE
kompose 0.2.0 (new formula)

### PR DESCRIPTION
**Pull request by blackstar257**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

**Comment by blackstar257** [comment]
Im not sure why the build failed. the bot doesn't show any information on the jenkins page

File: Formula/kompose.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class Compose2kube < Formula
```
Comment (MikeMcQuaid) [inline Formula/kompose.rb]: The class and filename are not in sync: https://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/8340/version=el_capitan/console

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

**Comment by MikeMcQuaid** [comment]
> Have you built your formula locally prior to submission with brew install <formula> (where <formula> is the name of the formula you're submitting)?

This definitely will not have worked locally.

File: Formula/kompose.rb (hunk 1 above)
Comment (blackstar257) [inline Formula/kompose.rb]: Apologies for wasting your time.

**Comment by blackstar257** [comment]
> Have you built your formula locally prior to submission with brew install (where is the name of the formula you're submitting)?
> This definitely will not have worked locally.

Accidentally committed the wrong file the first time around. It did work fine with the version that was living in 
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/kompose.rb

**Comment by blackstar257** [comment]
⇒  brew info kompose   
kompose: stable 64
Tool to move from `docker-compose` to Kubernetes
https://github.com/skippbox/kompose
/usr/local/Cellar/kompose/64 (3 files, 34.5M) *
  Built from source on 2016-10-05 at 09:11:27
From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/kompose.rb
⇒  kompose --version
kompose version 0.1.0 (8227684)

File: Formula/kompose.rb (hunk 1 above)
Comment (blackstar257) [inline Formula/kompose.rb]: I have since updated it.

File: Formula/kompose.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class Kompose < Formula
+  desc "Tool to move from `docker-compose` to Kubernetes"
+  homepage "https://github.com/skippbox/kompose"
+  url "https://github.com/skippbox/kompose/releases/download/v0.1.0/kompose_darwin-amd64.tar.gz"
```
Comment (bfontaine) [inline Formula/kompose.rb]: We need to build from source here.

File: Formula/kompose.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  sha256 "2cd1836fc0d5cd62bb72fb923cba98665c586087967b205b53dbcf2cdea9a9aa"
+
+  bottle :unneeded
+
+  def install
+    bin.install "kompose"
+  end
+
+  test do
+    system "#{bin}/kompose", "--version"
```
Comment (bfontaine) [inline Formula/kompose.rb]: Can’t we write a test that does something more substantial than e.g. `--help` or `--version`?

**Review by bfontaine — DISMISSED** [review]

**Comment by dunn** [comment]
ping re: the above comments 🔔

**Comment by blackstar257** [comment]
I should be able to get to this, this weekend.

**Comment by blackstar257** [comment]
Ill reopen if i get time to finish this.

**Comment by idcrook** [comment]
Is there something I can help with?

`brew search kompose`

```
No formula found for "kompose".
==> Searching pull requests...
Closed pull requests:
kompose 0.1.0 (new formula) (https://github.com/Homebrew/homebrew-core/pull/5276)
```

`brew cask search kompose`

    No Cask found for "kompose".

**Comment by blackstar257** [comment]
Attempted again.

**Comment by blackstar257** [comment]
@MikeMcQuaid 
@bfontaine 

Anything that you would like changed?

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format is `kompose 0.2.0 (new formula)`. Please complete the issue template in future PRs where this format is detailed in the linked [Contributing guidelines](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md). Thanks!

**Pull request closed without merging**